core/src/main/java/org/apache/calcite/sql/type/SqlTypeName.java

```diff
@@ -61,6 +61,7 @@
   FLOAT(PrecScale.NO_NO, false, Types.FLOAT, SqlTypeFamily.NUMERIC),
   REAL(PrecScale.NO_NO, false, Types.REAL, SqlTypeFamily.NUMERIC),
   DOUBLE(PrecScale.NO_NO, false, Types.DOUBLE, SqlTypeFamily.NUMERIC),
+  NUMBER(PrecScale.NO_NO, false, Types.NUMERIC, SqlTypeFamily.NUMERIC),
```

Comment:
Is it necessary to add this here? I don't see it being referenced anywhere else in the change.

Reply:
Ah good catch, I was just basing it off the VARBYTE one and forgot I didn't use it here. Will remove